src/bridge/cockpitrouter.c

```diff
@@ -709,11 +709,9 @@ static void
 process_kill (CockpitRouter *self,
               JsonObject *options)
 {
-  CockpitChannel *channel;
   GHashTableIter iter;
   const gchar *group = NULL;
   const gchar *host = NULL;
-  gpointer id, value;
   GList *list, *l;
 
   if (!cockpit_json_get_string (options, "group", NULL, &group))
```

Comment:
I didn't bother because we also don't protect against killing the special `"fence"` group. I couldn't imagine what such a check would be protecting against.

That said, I don't feel strong either and wouldn't mind adding it.